### PR TITLE
test: Fix 2 test files

### PR DESCRIPTION
**Pull request by Snidd111**
*Issue #, if available:*

*Description of changes:*
- When inspecting the data_sample.parquet file, it became apparent that values in it were corrupt or otherwise didn't match the csv equivalent. In order to allow for better cross input format testing, I'm replacing it with a file that matches the CSV equivalent. All columns are still classified as Strings.
- The 6column.json file also came up. It's not currently used as each test that would use it is using a programmatic schema. This fixes it for future use cases.

Here's the Parquet file for reference since it won't render:
Before
```
+---------+--------+--------------+----------+-----+-------------------+------+-----+--------------------+
|firstname|lastname|       address|      city|state|        phonenumber| title|level|               notes|
+---------+--------+--------------+----------+-----+-------------------+------+-----+--------------------+
|     John|   Smith|12345 Mills Rd|Charleston|   SC|       703-555-1234|   CEO|    X|                null|
|     Carl| Carlson|      0 Street|  Bethesda|   MD|        404-555-111|   CIO|   IX|This is a really ...|
|     Jane|  Kellog|      1 Two St|  New York|   NY|       304-555-1324|   COO|   IX|                null|
|     Mark|   James|        4 N St|  Richmond|   VA|       407-555-8888|    EA|  VII|        Secret notes|
|     Jake|Richards|  9 Hollows Rd|  Richmond|   VA|407-555-1222,SDE I"|    IV| null|                null|
|     Sean| Hendrix|  8 Hollows Rd|  Richmond|   VA|       407-555-4321| SDE I|   IV|Shana's younger b...|
|    Shana| Hendrix|  8 Hollows Rd|  Richmond|   VA|       407-555-4322|SDE II|    V| Sean's older sister|
+---------+--------+--------------+----------+-----+-------------------+------+-----+--------------------+
```

After
```
+---------+--------+--------------+----------+-----+------------+------+-----+--------------------+
|FirstName|LastName|       Address|      City|State| PhoneNumber| Title|Level|               Notes|
+---------+--------+--------------+----------+-----+------------+------+-----+--------------------+
|     John|   Smith|12345 Mills Rd|Charleston|   SC|703-555-1234|   CEO|   10|                null|
|     Carl| Carlson|      0 Street|  Bethesda|   MD| 404-555-111|   CIO|    9|This is a really ...|
|     Jane|  Kellog|      1 Two St|  New York|   NY|304-555-1324|   COO|    9|                null|
|     Mark|   James|        4 N St|  Richmond|   VA|407-555-8888|    EA|    7|        Secret notes|
|     Jake|Richards|  9 Hollows Rd|  Richmond|   VA|407-555-1222| SDE I|    4|                null|
|     Sean| Hendrix|  8 Hollows Rd|  Richmond|   VA|407-555-4321| SDE I|    4|Shana's younger b...|
|    Shana| Hendrix|  8 Hollows Rd|  Richmond|   VA|407-555-4322|SDE II|    5| Sean's older sister|
+---------+--------+--------------+----------+-----+------------+------+-----+--------------------+
```

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by m271828 — APPROVED** [review]

**Review by m271828 — DISMISSED** [review]
Latest main needed to be merged in.

**Comment by m271828** [comment]
@pnwamk You may need to approve this too. My approval is still here but it's requiring an approving review still after I merged in `main` so I could merge this for @Snidd111. My plan seems to have backfired.

**Comment by Snidd111** [comment]
rebased

**Review by pnwamk — DISMISSED** [review]

**Review by pnwamk — APPROVED** [review]

**Pull request merged**
